tools/pot/openvino/tools/pot/configs/hardware/cpu.json

```diff
@@ -225,7 +225,47 @@
         {
             "type": "Concat",
             "attributes": {
-                "scales": "unified"
+                "unified_scales": true
```

Comment:
Possible options:
|   | recurrent ops                                                    | other ops                            |
|---|------------------------------------------------------------------|--------------------------------------|
| 1 | "unified_scales": [[0,1], [3,4]]                                 | "unified_all_scales": true           |
| 2 | "unified_scales": "[[0, 1], [3, 4]]",                            | "unified_scales": "all"              |
| 3 | "unified_scales": [[0, 1], [3, 4]],                            | "unified_scales": "all"              |
| 4 | "unified_scales" : { "num_groups": 2, "groups": [[0,1], [3,4]] } | "unified_scales" : {"num_groups": 1} |

Reply:
I didn't notice the difference between 2nd and 3rd but vote for them :-)

Reply:
2nd: "unified_scales" -> **str** for both cases
3rd: "unified_scales" -> **list** for recurrent and **str** otherwise

Reply:
I'd prefer 3rd option

Reply:
ok, so do I. Let's come up with the 3rd one.